autorag_client/README.md

```diff
@@ -0,0 +1,27 @@
+# AutoRAG Client
+
+This is the client for AutoRAG Studio Cloud.
+
+To register this, please contact to Jeffrey Kim. (jeffrey@auto-rag.com)
```

Comment:
Need to change the e-mail address to martin@auto-rag.com

Reply:
Ok